pom.xml

```diff
@@ -26,9 +26,36 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.uber.tchannel</groupId>
-    <artifactId>tchannel</artifactId>
+    <artifactId>tchannel-java</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
```

Comment:
with this versioning scheme only 2 release are allowed per version, the first one the SNAPSHOT or developer release and the next is 0.1.0 which is the actual release.